.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vendor/mozilla-rust-sdk/googleapis-raw/grpc"]
+	path = vendor/mozilla-rust-sdk/googleapis-raw/grpc
+	url = https://github.com/grpc/grpc.git
```

Comment:
it [looks like](https://stackoverflow.com/questions/49490272/how-to-specify-a-tag-in-gitmodules-file) you may be able to specify a tag here. Would it be possible to do this here so we know which specific version of the grpc lib we're depending on?

Reply:
Thanks, I'll see if I can dig into that. Part of the problem is also that dependbot will flag "old" versions as well. 

I honestly wonder if part of the problem is also home bound folks "catching up on bugs" and releasing a LOT of new revisions of libraries causing all sorts of fun cascades.

Reply:
no longer an issue with the update of the vendored code.